client/state/action-types.js

```diff
@@ -498,6 +498,9 @@ export const THEME_CLEAR_ACTIVATED = 'THEME_CLEAR_ACTIVATED';
 export const THEME_DETAILS_REQUEST = 'THEME_DETAILS_REQUEST';
 export const THEME_DETAILS_RECEIVE = 'THEME_DETAILS_RECEIVE';
 export const THEME_DETAILS_RECEIVE_FAILURE = 'THEME_DETAILS_RECEIVE_FAILURE';
+export const TICKET_SUPPORT_CONFIGURATION_REQUEST = 'TICKET_SUPPORT_CONFIGURATION_REQUEST';
+export const TICKET_SUPPORT_CONFIGURATION_REQUEST_SUCCESS = 'TICKET_SUPPORT_CONFIGURATION_REQUEST_SUCCESS';
+export const TICKET_SUPPORT_CONFIGURATION_REQUEST_FAILURE = 'TICKET_SUPPORT_CONFIGURATION_REQUEST_FAILURE';
```

Comment:
This is related to my other comment about changing the location in the state tree; how about `HELP_TICKET_CONFIGURATION_*` for actions? That would help us keep more of the help related stuff together.